rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/SubmitJobCommand.java

```diff
@@ -140,8 +146,8 @@ private Boolean isFileExisting(String pathname) {
     }
 
     private Boolean isFilePathValid(String pathname) {
-        String regex = "^(\\/)?([^\\/\\\u0000]+(\\/)?)+\\.xml";
-        final Pattern pattern = Pattern.compile(regex);
+        String correctPathnameRegex = "^(?>[a-z]:)?(?>\\\\|\\/)?([^\\\\\\/?%*:|\\\"<>\\r\\n]+(?>\\\\|\\/)?)+$";
```

Comment:
It is up to you to choose, either regex, or a more readable solution

public static boolean isValidPath(String path) {

            try {

                Paths.get(path);

            } catch (InvalidPathException | NullPointerException ex) {
                return false;
            }

            return true;
        }